sdk-base/src/main/java/com/mapbox/maps/plugin/annotation/ClusterOptions.kt

```diff
@@ -0,0 +1,69 @@
+package com.mapbox.maps.plugin.annotation
+
+import android.graphics.Color
+
+/**
+ * Options to show and configure symbol clustering with using SymbolManager.
+ * <p>
+ * It exposes a minimal of configuration options, a more advanced setup can be created manually with
+ * using CircleLayer and SymbolLayers directly.
+ * </p>
+ */
+data class ClusterOptions(
```

Comment:
The previous version of this file that worked with v9.0, we were able to use expressions to style. Would we be able to support this as well? https://github.com/mapbox/mapbox-plugins-android/blob/cb8ba99b74938508fe5a572b93dc75485e17afd0/plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/ClusterOptions.java#L42-L49

Reply:
This definition is in sdk-base and Expressions are in style extension, I guess we are unable to support this unless we could find a way to resolve this dependency

Reply:
we could expose them as expression json String. I have added a `Value.toJson()` extension function to convert the constructed Expression to json string in sdk-base.